src/jsifier.js

```diff
@@ -299,7 +300,7 @@ function JSify(data, functionsOnly) {
         });
       });
       if (VERBOSE) printErr('adding ' + finalName + ' and deps ' + deps + ' : ' + (snippet + '').substr(0, 40));
-      var depsText = (deps ? '\n' + deps.map(addFromLibrary).filter(function(x) { return x != '' }).join('\n') : '');
+      var depsText = (deps ? '\n' + deps.map(function(dep){ addFromLibrary(dep, ident + "__deps: ['" + deps.join("','")+"']")}).filter(function(x) { return x != '' }).join('\n') : '');
```

Comment:
this could go further and add `dependent + ' => ' + (your current code)` so that it shows the entire chain, all the way from the beginning, by appending at each step? maybe excessive though, could be slower.

(if we do that then `dependent` might be `dependents` or `depsInfo` or such)